transit_odp/timetables/tests/test_extract_metadata.py

```diff
@@ -0,0 +1,110 @@
+import os
+from datetime import datetime
+from unittest import skip
+
+import pandas as pd
+import pytest
+from dateutil import tz
+from dateutil.parser import parse
+from ddt import data, ddt, unpack
+from django.contrib.gis.geos import Point
+from django.core.files import File
+from django.test import TestCase
+from unittest.mock import patch, MagicMock
+
+
+from transit_odp.naptan.factories import AdminAreaFactory
+from transit_odp.naptan.models import AdminArea, District, Locality, StopPoint
+from transit_odp.organisation.constants import FeedStatus
+from transit_odp.organisation.factories import (
+    DatasetRevisionFactory,
+    OrganisationFactory,
+)
+from transit_odp.pipelines.factories import DatasetETLTaskResultFactory
+from transit_odp.timetables.extract import TransXChangeExtractor
+
+from transit_odp.xmltoolkit.xml_toolkit import XmlToolkit
+
+TZ = tz.gettz("Europe/London")
+EMPTY_TIMESTAMP = None
+
+
+class ExtractBaseTestCase(TestCase):
```

Comment:
Can `ExtractBaseTestCase` be reused from `transit_odp/pipelines/tests/test_dataset_etl/test_extract_metadata.py`

Reply:
new file is created for extracting test metadata as it is referring the correct extractor